packages/matter-node.js-examples/src/examples/DeviceNode.ts

```diff
@@ -269,12 +270,18 @@ class Device {
                             request: { ssid, credentials, breadcrumb },
                             attributes: { lastNetworkingStatus, lastNetworkId },
                             endpoint,
+                            session,
                         }) => {
                             console.log(
                                 `---> addOrUpdateWiFiNetwork called on NetworkCommissioning cluster: ${ssid.toHex()} ${credentials.toHex()} ${breadcrumb}`,
                             );
 
-                            // TODO Check if failsafe is armed
+                            if (!session.getContext().isFailsafeArmed()) {
```

Comment:
In fact it is nothing of the session, it is the "device" ... and the session is the place to get to the context ... so it would be a "assertFailsafeArmed(session)" which also feels wrong because it lets assume it is the session, you know what I mean?

Reply:
PS: and in fact this "session.getContext" we have inmany ckluster servers too ... as soon as we need to do something in the context of "the device". This is one topic were I hope to find a better idea

Reply:
`session.getContext().assertFailSafeArmed()` is fine too, point 1 was regarding if statement & following lines that're in this file.

Note point 2 also, that's probably more important

Reply:
Re 1.) Ahhh now I get you ... added and yes makes a lot sense :-)
Re 2.) I moved it into an extra file ... 

Check last two commit please ... wdyt?

Reply:
LGTM